net-analyzer/bro/bro-2.4.1-r2.ebuild

```diff
@@ -44,6 +44,12 @@ pkg_setup() {
 	use python && python-single-r1_pkg_setup
 }
 
+src_prepare() {
+	# Disable shell script feature which sets interpreter to build time default
+	grep -rl 'set(_regex "^#!\[ \]\*/usr/bin/env' . | xargs sed -i -e '/set(_regex \"^#!\[ \]\*\/usr\/bin\/env/,+17d' || die
```

Comment:
Please don't:
1. use `xargs`,
2. do `sed` with ranges that aren't clearly defined.

This thing really asks for accidental mis-sed on version bump. So, a patch would probably be better, unless you can figure out another way of doing this.

Also, if the result of this is unversioned `python` shebang, then this is wrong and you should fix it to store correct path instead.